src/molecules/Tabs/TabButton.native.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { View, TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+import Flex from '../../atoms/Flex';
+import Text from '../../atoms/Text';
+import { getColor, makePxValue } from '../../_helpers/theme';
+import Icon from '../../atoms/Icon';
+import Space from '../../atoms/Space';
+import automation from '../../_helpers/automation-attributes';
+import isEmpty from '../../_helpers/isEmpty';
+import { getIconNames } from '../../_helpers/icon';
+
+const styles = {
+  label: {
+    borderBottomColor({ active, theme, disabled }) {
+      if (disabled) {
+        if (active) {
+          return getColor(theme, 'primary.930');
+        }
+      }
+      if (active) {
+        return getColor(theme, 'primary.800');
+      }
+      return 'transparent';
+    },
+    color({ active, disabled, pressed }) {
+      if (pressed) {
+        if (active) {
+          return 'primary.900';
+        }
+        return 'shade.970';
+      } else if (active) {
+        return disabled ? 'primary.700' : 'primary.800';
+      } else {
+        return disabled ? 'shade.940' : 'shade.960';
+      }
+    },
+  },
+  tab: {
+    backgroundColor({ theme, active, pressed }) {
+      if (pressed) {
+        if (active) {
+          return getColor(theme, 'primary.920');
+        }
+        return getColor(theme, 'tone.940');
+      }
+      return 'transparent';
+    },
+  },
+};
+
+const StyledTabButton = styled(TouchableOpacity)`
+  min-width: ${makePxValue(6)};
+  background-color: ${styles.tab.backgroundColor};
+  border-top-left-radius: ${(props) => props.theme.spacings.xxsmall};
+  border-top-right-radius: ${(props) => props.theme.spacings.xxsmall};
+`;
+
+const Label = styled(View)`
+  border-bottom-width: 1px;
+  border-bottom-color: ${styles.label.borderBottomColor};
+`;
+
+const TabButton = ({ label, active, icon, onPress, disabled, testID }) => {
+  const [pressed, setPressed] = useState(false);
+
+  const onPressIn = useCallback(() => {
+    setPressed(true);
+  }, []);
+
+  const onPressOut = useCallback(() => {
```

Comment:
`handlePressOut`?

Reply:
I think shridhar kept it like that to match what's below

```
<StyledTabButton
        activeOpacity={1}
        disabled={disabled}
        onPress={onPress}
        onPressIn={onPressIn}
        onPressOut={onPressOut}
```